docs/SPIR-V.rst

```diff
@@ -1701,15 +1701,21 @@ The SPIR-V ``InvocationID`` (``SV_OutputControlPointID`` in HLSL) is used to ind
 into the InputPatch and OutputPatch arrays to read/write information for the given
 vertex.
 
+The hull main entry function in HLSL returns only one value (say, of type ``T``), but 
+that function is in fact executed several times. Therefore, we create a stage output variable
```

Comment:
... executed once for each control point. The Vulkan spec requires that "Tessellation control shader per-vertex output variables and blocks, and tessellation control, tessellation evaluation, and geometry shader per-vertex input variables and blocks are required to be declared as arrays, with each element representing input or output values for a single vertex of a multi-vertex primitive." Therefore, we need to create ...